spec/services/discovery_engine/quality/evaluation_spec.rb

```diff
@@ -11,12 +11,21 @@
   let(:operation) { double("operation", error?: false, wait_until_done!: true, results: response) }
   let(:response) { double("response", name: "/evaluations/1") }
   let(:google_time_stamp) { double("google_time_stamp", nanos: 812_173_000, seconds: 1_753_600_645) }
+  let(:search_request_response) do
```

Comment:
The pattern in this file is for the stub name to match the string passed to the `double()` method, ie we'd have `let(:search_request) {double("search_request"...` and the same for the `evaluation_spec`